--- a/setup.py
+++ b/setup.py
@@ -459,7 +459,7 @@ def get_mupdf_internal(out, location=None, local_tgz=None):
         location = f'https://mupdf.com/downloads/archive/mupdf-{version_mupdf}-source.tar.gz'
         #location = 'git:--branch master https://github.com/ArtifexSoftware/mupdf.git'

-    if location == '':
+    if location in ('', 'system'):
         # Use system mupdf.
         return None, location

